app/styles/components/ilios-navigation.scss

```diff
@@ -109,9 +112,10 @@
     height: 100%;
     text-decoration: none;
     width: 100%;
+    font-weight: 400;
 
     &.active {
-      font-weight: bold;
+      font-weight: 700;
```

Comment:
i'm curious about this change. why go to numbers here?

Reply:
Habit. We pre load 400,600, and 700 weights of our font so sticking to those numbers avoids needing to load new font files. I can never remember what weight each keyword represents.